package.json

```diff
@@ -21,21 +21,23 @@
     "url": "https://github.com/meetup/meetup-web-mocks/issues"
   },
   "homepage": "https://github.com/meetup/meetup-web-mocks#readme",
-  "dependencies": {
+  "peerDependencies":{
+    "react": "^15.3.2",
```

Comment:
We can't use `^` notation ranges, because it doesn't cover major version updates. In these changes, `react-redux` and `react-router` are already at higher major versions and this will introducing a peer-dependency error on install.